src/components/view/ConnectEhrView/ConnectEhrView.tsx

```diff
@@ -32,7 +32,7 @@ export default function (props: ConnectEhrViewProps) {
                 <ExternalAccountsPreview previewState={props.preview ? "Default" : undefined} applicationUrl={props.externalAccountsApplicationUrl} excludeDeviceManufacturers={true}/>
             </Card>
             <Card>
-                <ProviderSearch previewState={props.preview ? "Default" : undefined} providerCategories={["Provider", "Health Plan"]}/>
+                <ProviderSearch previewState={props.preview ? "Default" : undefined} providerCategories={["Provider", "Health Plan"]} pageSize={10} />
```

Comment:
Would default to 100 here - that is actually plenty fast and requires less waiting while you scroll for the next page

Reply:
Actually I am not even sure it makes sense to have it be a parameter...I would just hardcode pageSize as 100 inside the component and be done with it.